src/ngraph/runtime/cpu/pass/cpu_fusion.hpp

```diff
@@ -66,6 +66,8 @@ class ngraph::runtime::cpu::pass::CPUFusion : public ngraph::pass::GraphRewrite
             construct_conv_bias_bprop();
             construct_conv_bias_folded_batch_norm();
             construct_conv_bias_affine_folding();
+            construct_groupconv_batchnorm_global_stats_folding(); //must be called before BNRelu
```

Comment:
I don't know if this comment relevant anymore since we aren't fusing GroupConv with BoundedRelus